README.md

```diff
@@ -41,7 +41,7 @@ which Pando subscribes.
 #### Prerequisite
 Pando accepts IPLD data from metadata providers with the following required IPLD children nodes:
 1. `Provider String` - provider's peer ID
-2. `Signature Bytes` - signature of the IPLD data: sign( bytes(PreviousID) . bytes(Payload) . bytes(Provider)) )
+2. `Signature Bytes` - signature of the IPLD data: dagjson.Encode( ipld.Node{PreviousID, Provider, Payload} ).Bytes()
```

Comment:
sig, err := SignWithPrivky(signKey, meta)
	if err != nil {
		return nil, err
	}

	// Add signature
	meta.Signature = sig

Signature Bytes is bytes from envelop's Marshal(). The envelop will use `metaSignatureRecord`, PrivateKey and bytes from IPLD node(dagjson.Encode)

// Signs metadata using libp2p envelope
func SignWithPrivky(privkey crypto.PrivKey, meta *Metadata) ([]byte, error) {
	metaID, err := signMetadata(meta)
	if err != nil {
		return nil, err
	}
	envelope, err := record.Seal(&metaSignatureRecord{metaID: metaID}, privkey)
	if err != nil {
		return nil, err
	}
	return envelope.Marshal()
}